aiida/cmdline/params/types/test_module.py

```diff
@@ -12,35 +12,9 @@
 
 
 class TestModuleParamType(click.ParamType):
-    """Parameter type to represent a unittest module."""
+    """Parameter type to represent a unittest module.
 
-    name = 'test module'
-
-    @staticmethod
-    def get_test_modules():
-        """Returns a list of known test modules."""
-        from aiida.backends.tests import get_db_test_names
-
-        prefix_db = 'db'
-        modules_db = get_db_test_names()
-        modules_base = ['aiida.cmdline', 'aiida.common', 'aiida.schedulers', 'aiida.transports']
-
-        test_modules = {}
-
-        for module in modules_base:
-            test_modules[module] = None
+    Defunct - remove when removing the "verdi devel tests" command.
```

Comment:
Maybe add the deprecation warning using `v2.0.0` because that way it is easy to search the code base when removing deprecated stuff

Reply:
done!